.github/workflows/create-release-pr.yml

```diff
@@ -41,16 +23,65 @@ on:
         required: false
 
 jobs:
+  resolve-bases:
+    # Determines if the release is hotfix or not based on semver. Then sets the appropriate base branches for the checkout and release PR.
+    runs-on: ubuntu-latest
+    outputs:
+      checkout_base: ${{ steps.out.outputs.checkout_base }}
+      release_base: ${{ steps.out.outputs.release_base }}
+    steps:
+      - id: out
+        shell: bash
+        env:
+          SEMVER: ${{ inputs.semver-version }}
+        run: |
+          set -euo pipefail
+
+          echo "Provided semver input: ${SEMVER}"
+          # Validate semver
+          if ! [[ "${SEMVER:-}" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo "Error: semver-version must be numeric X.Y.Z, got: ${SEMVER:-<empty>}" >&2
+            exit 1
+          fi
+
+          patch="${SEMVER##*.}"
+          if [ "$patch" -gt 0 ]; then
+            echo "Detected hotfix (patch=$patch): using stable for both bases."
```

Comment:
For consistency with the other branch of the "if" condition, can we replace the line by:
````
echo "Detected non-hotfix (patch=0): using stable for checkout and stable for release PR base."
````